### PR TITLE
[#1254] Rework Column component

### LINKED ISSUE #1254: Columns go off the page
Taiga 334

If too many columns are specified, then they go off the page
Screenshot from 2022-01-27 14-32-43

### PR DESCRIPTION
**Pull request by SilviaAmAm**
Fixes #1254 

Front end: https://github.com/open-formulieren/open-forms-sdk/pull/163

### CODE REVIEW

**Comment by SilviaAmAm** [comment]
Current look of it:

Editor:
Screenshot from 2022-02-11 16-47-28

SDK:
Screenshot from 2022-02-11 16-47-38

File: src/openforms/js/components/formio_builder/templates/columns.ejs
Comment (sergei-maertens) [inline src/openforms/js/components/formio_builder/templates/columns.ejs:3]:
```suggestion
        <div class="col col--span-{{column.size}}" ref="{{ctx.columnKey}}">
```
proper BEM class naming :-)

File: src/openforms/scss/components/builder/_columns.scss
Comment (sergei-maertens) [inline src/openforms/scss/components/builder/_columns.scss:71]: https://sass-lang.com/documentation/at-rules/control/each :-)

**Review by sergei-maertens — CHANGES_REQUESTED** [review]

File: src/openforms/scss/components/builder/_columns.scss
Comment (SilviaAmAm) [reply src/openforms/scss/components/builder/_columns.scss]: Amazing :scream:

**Review by sergei-maertens — APPROVED** [review]

**Comment by sergei-maertens** [comment]
Waiting for SDK to be approved before we merge this :)

**Pull request merged**
